app/src/main/res/values/colors_migrating.xml

```diff
@@ -112,4 +112,5 @@
   <color name="admin_highlighted">#A5D3EC</color>
   <color name="coming_soon_end">#674172</color>
   <color name="coming_soon_start">#7659B6</color>
+  <color name="spotlightBackground">#BF000000</color>
```

Comment:
Nit: please name according to snake_case naming standards.

Reply:
done

Reply:
@JishnuGoyal This is still not solved. (Maybe because I started code review like ~25 mins back so the commit was made after that)

Reply:
yes, this was resolved, thanks!